Makefile

```diff
@@ -55,6 +55,7 @@ errdoc:tools/bin/errdoc-gen
 lint:tools/bin/revive
 	@echo "linting"
 	@tools/bin/revive -formatter friendly -config tools/check/revive.toml $(FILES_TIDB_TESTS)
```

Comment:
In fact, we can enable ```revive``` for all br package. you only need to change ```build/nogo_config.json``` to add ```br``` package into ```only_files``` of ```revive```.